### ISSUE TITLE
Startup delay on windows 10

### ISSUE THREAD

**Issue by kostadinnm** [martinrotter/textosaurus#54]
There is a noticeable delay when starting textosaurus on latest windows 10(1803 build 17134.345). In comparison - on a windows 7(virtualized over the windows 10 machine) the loading is instantaneous. I tried both a portable and an installed version of textosaurus. No resource is overloaded in task manager - it just doesn't start immediately. It takes a little more than 20 seconds on my ssd-based windows 10 workstation. Tried adding textosaurus as a trusted program in windows 10 defender, but it doesn't help.

Just tried an old virtual windows 10 1607 which is fine - loads as fast as windows 7. I guess there are some changes in between that are hindering the load times of some qt-apps...

Cheers,
Kostadin

**Comment by martinrotter** [comment]
Can you btw provide [debug log file](https://github.com/martinrotter/textosaurus/wiki/Generating-debug-log-file)?

**Comment by kostadinnm** [comment]
Last night it suddenly got ok on the loading side. Today I tried to capture it with some screen recording on the other physical machine, but the worse it gets there(2 core cpu, plenty of ram, 10K rpm hdd) is a little above 10 seconds. On the main one(4 core cpu, plenty of ram, ssd) it's instantaneous. I was about to respond last night, but github's public site had been down for a while.
I suspect there's undergoing inspection by the OS, which I don't mind. I just wish that profiling is more evident so that there are no false-positives like this.
I'm closing this one as well.

Best regards,
Kostadin

**Comment by kostadinnm** [comment]
Last night it suddenly got ok on the loading side. Today I tried to capture it with some screen recording on the other physical machine, but the worse it gets there(2 core cpu, plenty of ram, 10K rpm hdd) is a little above 10 seconds. On the main one(4 core cpu, plenty of ram, ssd) it's instantaneous. I was about to respond last night, but github's public site had been down for a while.
I suspect there's undergoing inspection by the OS, which I don't mind. I just wish that profiling is more evident so that there are no false-positives like this.
I'm closing this one as well.

Best regards,
Kostadin

**Comment by kostadinnm** [comment]
Last night it suddenly got ok on the loading side. Today I tried to capture it with some screen recording on the other physical machine, but the worse it gets there(2 core cpu, plenty of ram, 10K rpm hdd) is a little above 10 seconds. On the main one(4 core cpu, plenty of ram, ssd) it's instantaneous. I was about to respond last night, but github's public site had been down for a while.
I suspect there's undergoing inspection by the OS, which I don't mind. I just wish that profiling is more evident so that there are no false-positives like this.
I'm closing this one as well.

Best regards,
Kostadin

**Comment by kostadinnm** [comment]
Last night it suddenly got ok on the loading side. Today I tried to capture it with some screen recording on the other physical machine, but the worse it gets there(2 core cpu, plenty of ram, 10K rpm hdd) is a little above 10 seconds. On the main one(4 core cpu, plenty of ram, ssd) it's instantaneous. I was about to respond last night, but github's public site had been down for a while.
I suspect there's undergoing inspection by the OS, which I don't mind. I just wish that profiling is more evident so that there are no false-positives like this.
I'm closing this one.

Best regards,
Kostadin

**Comment by kostadinnm** [comment]
Last night it suddenly got ok on the loading side. Today I tried to capture it with some screen recording on the other physical machine, but the worse it gets there(2 core cpu, plenty of ram, 10K rpm hdd) is a little above 10 seconds. On the main one(4 core cpu, plenty of ram, ssd) it's instantaneous. I was about to respond last night, but github's public site had been down for a while.
I suspect there's undergoing inspection by the OS, which I don't mind. I just wish that profiling is more evident so that there are no false-positives like this.

Best regards,
Kostadin

**Comment by kostadinnm** [comment]
This gets frustrating

**Comment by martinrotter** [comment]
` is a little above 10 seconds` This is still quite bad result. I would expect Textosaurus to start almost instantly on all PCs. It is quite lightweight app. Nothing expensive happens during its startup.

**Comment by kostadinnm** [comment]
Well, that's what it is, but if you want I could make a video screen capture. It's far better than yesterday and the program window does show up in around the 5th second with a blank screen, then between 10-12 seconds it's fully loaded. On the main station is as fast as a bullet, though.

**Comment by martinrotter** [comment]
> Well, that's what it is, but if you want I could make a video screen capture. It's far better than yesterday and the program window does show up in around the 5th second with a blank screen, then between 10-12 seconds it's fully loaded. On the main station is as fast as a bullet, though.

That is very very slow startup. I just upgraded to latest win10 build and application loads instantly. For example on Linux, app loads incredibly fast, "instantly". Maybe, if you provide debug log (as I wrote earlier) we could identify the area of code which is responsible for delays. It is just weird.

What if you disconnect from network and try to launch app? Is it faster? What if you run it as admin? It makes really no sense why it should be slower on some win build and fast on another. It must be some kind of interference.
